src/donation.html

```diff
@@ -0,0 +1,116 @@
+---
+layout: nofooter.html
+pageTitle: Donation
+tags: nav
+order: 5
+---
+<h1 class="page-title">Donation</h1>
+
+<ul class="donation-sub">
+    <li>Drop-off donations time:</li>
+    <li>9Am-5Pm Thursday - Sunday</li>
+    <li>Pick-ups time:</li>
+    <li>7Am-2Pm Monday - Friday</li>
+</ul>
+
+<ul class="donation-sub">
+    <li>Call us: 410-385-1700</li>
+    <li>Email: donations@secondchanceinc.org</li>
+</ul>
+
+<h2>Donation Form</h2>
+<section class="form">
+    <label for="name">Name</label>
+    <input id="name" type="text" placeholder="Required">
+
+    <label for="phone-number">Phone Number</label>
+    <input id="phone-number" type="tel" placeholder="Required">
+
+    <p class="type-of-donation">Type of donation</p>
+    <label for="donate-items">
+        <input id="donate-items" type="checkbox">Donate Items</label>
+    <label for="donate-items">
+        <input id="donate-a-house" type="checkbox">Donate a House</label>
+    <label for="donate-a-house">
```

Comment:
Note that these two labels have confused `for` attributes:
```suggestion
    <label for="donate-a-house">
        <input id="donate-a-house" type="checkbox">Donate a House</label>
    <label for="donate-money">
```